File: src/main/java/leets/leenk/domain/leenk/application/exception/AlreadyParticipatedException.java
package leets.leenk.domain.leenk.application.exception;

import leets.leenk.global.common.exception.BaseException;

public class AlreadyParticipatedException extends BaseException {
    public AlreadyParticipatedException() {
        super(ErrorCode.LEENK_ALREADY_PARTICIPATED);
    }
}

File: src/main/kotlin/leets/leenk/domain/leenk/application/exception/AlreadyParticipatedException.kt
package leets.leenk.domain.leenk.application.exception;

import leets.leenk.global.common.exception.BaseException;

public class AlreadyParticipatedException extends BaseException {
    public AlreadyParticipatedException() {
        super(LeenkErrorCode.LEENK_ALREADY_PARTICIPATED);
    }
}
